pkg/agent/openflow/client.go

```diff
@@ -582,6 +598,7 @@ func (c *client) InstallClusterServiceFlows() error {
 		flows = append(flows, c.serviceHairpinResponseDNATFlow(binding.ProtocolIPv6))
 		flows = append(flows, c.serviceLBBypassFlows(binding.ProtocolIPv6)...)
 	}
+
```

Comment:
Please try to avoid such changes unless you are changing related code. I can see PRs add empty lines and delete empty lines. Not sure by accident or somebody prefers having more empty lines while somebody not, but it's a bit distracting. As long as the code has been merged, let's leave it as is.

Reply:
Got it